contracts/cw4-stake/src/state.rs

```diff
@@ -31,3 +33,44 @@ pub const MEMBERS: SnapshotMap<&[u8], u64> = SnapshotMap::new(
 );
 
 pub const STAKE: Map<&[u8], Uint128> = Map::new("stake");
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug, Default)]
+pub struct GenericBalance {
```

Comment:
This makes sense for another usecase with multiple possibilities.

Here the know we have a balance of exactly one token type. This could be a simpler enum (and never Vec<Cw20Coin>, but just one). In the end, you could likely just use Balance where you want to use this.